--- mesa-21.3.9/src/amd/vulkan/winsys/null/radv_null_winsys.c
+++ mesa-21.3.9/src/amd/vulkan/winsys/null/radv_null_winsys.c.new
@@ -41,6 +41,10 @@
    [CHIP_OLAND] = {0x6060, 2, true},
    [CHIP_HAINAN] = {0x6660, 2, true},
    [CHIP_BONAIRE] = {0x6640, 4, true},
+   [CHIP_LIVERPOOL] = {0x9920, 2, false},
+   [CHIP_LIVERPOOL] = {0x9922, 2, false},
+   [CHIP_LIVERPOOL] = {0x9923, 2, false},
+   [CHIP_GLADIUS] = {0x9924, 4, false},
    [CHIP_KAVERI] = {0x1304, 2, false},
    [CHIP_KABINI] = {0x9830, 2, false},
    [CHIP_HAWAII] = {0x67A0, 16, true},
